qualys-web-application-scanner/operations.py

```diff
@@ -39,12 +55,18 @@ def make_rest_call(self, endpoint, params=None,
                                         params=params)
             if response.ok:
                 if response.status_code == 200:
-                    return response.json()
+                    if is_file_action:
+                        return response.content
+                    else:
+                        return response.json()
             # return json.loads(response.content.decode('utf-8')) if response.content.decode('utf-8') else ''
```

Comment:
Remove commented code

Reply:
Lets keep it as this is uncertified connector if something breaks may have to revert back.